src/config/dynamic-theme-fixes.config

```diff
@@ -11532,6 +11532,23 @@ p.site-title
 
 ================================
 
+planetminecraft.com
+
+CSS
+#container,
+.forum_replies_container,
+.forum_reply,
+.contents,
+.spoiler_header,
+#resource-log,
+.tag,
+.view_more:not(:hover),
+#popup {
+    background: var(--darkreader-bg--content-bg) !important;
```

Comment:
I feel that a bit too much selectors are using this specific variables. E.g. `#container` should be using `--darkreader-neutral-bg`.

Reply:
makes sense, I don't think there is a reason for any of those to be a specific color, so I used neutral-background for all of them now.